dotnet/Razorvine.Pyrolite/Tests/Pickle/UnpickleComplexTests.cs

```diff
@@ -89,7 +89,7 @@ private static object U(byte[] data)
 		UnpickleRealProxy(pickledProxy);
 	}
 
-	[Fact]
+	[Fact(Skip = "Failing with \"insecure string pickle\"")]
```

Comment:
> This test is not supposed to fail, and the skip should not be here. Why was this added?

It fails for me without any changes; I just clone the repo, build, run the tests, and it fails.
![image](https://user-images.githubusercontent.com/2642209/56242440-4a3db700-6066-11e9-86e3-4c893702aed7.png)

Reply:
That's odd. It works fine on my system (Linux). Seems like it's a windows specific thing, I'll try to look a bit into this but I don't build this library on Windows..

Reply:
I debugged it. The issue is the difference between "\r\n" and "\n" line endings.  load_string is calling PickleUtils.readline, which strips off the "\n" but not the "\r" before it on Windows.  It then checks whether the string ends with the same quote character that the line begins with, and it fails because it sees the last character as '\r' instead of '"'.

Reply:
thanks for the info I'll try to fix it for windows later.